### PR TITLE
Add an advice / rumour -module, giving useful information on the BBS

### PR DESCRIPTION
**Pull request by impaktor**
I did this, since when I add @clausimu's pirate branch, I want hints to where to find pirate stations. At least the ones packing the special features I plant to add. More on that another time.

Oh, dear. I still haven't said what this is. It just puts some word of advice on the BBS. No charge. Keep it simple. There are 10 flavours. In three versions (Traveller's advice, Traveller's tale, and Rumour).

For now, the advice/rumour/tale is specific to the station. Thinking is: on
that particular station someone lives, or some meme/information/rumour has
taken hold, and in the spirit of sharing this is freely posted on the BBS.

The advice is some "lore" or information that adds useful information
on how to play the game, or things to discover, or make use of in game play. I tried to word it as some profound piece of wisdom.

Ideally, this information would "leak" from some other more relaxed social
setting, like a bar, from more experienced pilots / bartender.

## TO DO before merge:
- [x] BBS Icon needed (a mouth, or a speaking bubble?)
- [x] I need to re-write some of the flavouts. Especially `ADVICE_4_BODYTEXT`

### CODE REVIEW

**Comment by impaktor** [comment]
Forgot to add "1000 **words" (i.e. picture):
2017-04-19-122601_823x189_scrot

Also, "Traveller's advice" have inflated index numbers, to give the illusion there are many. 
Like Star Wars IV.

**Comment by fluffyfreak** [comment]
I like

**Comment by impaktor** [comment]
OK, this is done, except for me discovering bad/wrong API documentation for `rand:Number`, and then the code in that file doesn't start until way down, which confuses my auto-detect-indentation mode (it looks at first X lines), so it thinks tabs are 3 char long, and got a bit "creative" --- I'll fix it.

**Comment by impaktor** [comment]
I was wrong about being wrong on the indentation. Two tabs are two tabs are two tabs. Even though my editor thinks they're 3 char width. Not sure why those lines in LuaRand.cpp were badly indented before. Git blame blames me and rob, if I read it right. 

Anyway, I'm finished now. I also added a tenth advice (relocate faction when heat it up), and increased spawn probability to 20% per BBS for an "advert".

Hey, maybe @clausimu wants to review?

File: data/lang/module-advice/en.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      "description" : "",
+      "message" : "Take note of which local business to trust or not, upon completing a transaction outside the confines of the law."
+   },
+   "ADVICE_2_HEADLINE" : {
+      "description" : "(A reference to the famous poem The Road Not Taken, by Robert Frost)",
+      "message" : "The Road Faster Taken"
+   },
+   "ADVICE_2_BODYTEXT" : {
+      "description" : "",
+      "message" : "Many small distances are faster travelled than their sum"
```
Comment (clausimu) [inline data/lang/module-advice/en.json]: Period missing at the end of the sentence.

File: data/libs/SpaceStation.lua [hunk 2]
```diff
@@ -441,7 +441,8 @@ SpaceStation.adverts = {}
 --
 --   description - text to display in the bulletin board
 --
---   icon - (option) filename of an icon to display alongside the advert
+--   icon - optional, filename of an icon to display alongside the advert.
+--          Defualuts to bullet (data/icons/bbs/default).
```
Comment (clausimu) [inline data/libs/SpaceStation.lua]: "Defualuts" => "Defaults"

File: data/lang/module-advice/en.json (hunk 1 above)
Comment (impaktor) [inline data/lang/module-advice/en.json]: fixed, thanks!

File: data/libs/SpaceStation.lua (hunk 2 above)
Comment (impaktor) [inline data/libs/SpaceStation.lua]: ...and fixed

**Comment by impaktor** [comment]
@clausimu thanks! I fixed and pushed the errors you spotted.

**Comment by clausimu** [comment]
Overall, nice idea! Some of the advice reads like a game play tutorial, which may kill immersion for players a little. I guess we can move these bits once in-game help is available (didn't @ecraven want to show wiki pages or something?) and extend with game story rumors instead. But generally a good addition to the game.

**Comment by impaktor** [comment]
> Some of the advice reads like a game play tutorial, which may kill immersion for players a little

Yeah, that's what I wanted / tried  to avoid. If you have suggestions for how to improve, I'm all ears.

I separated the "TRAVELLER'S ADVICE" pre-headline to separate strings, so the translated advices are easier to lift out and move elsewhere in the future.

**Pull request merged**